ci/docker/install/ubuntu_scala.sh

```diff
@@ -23,9 +23,8 @@
 set -ex
 # install libraries for mxnet's scala package on ubuntu
 apt-get install -y software-properties-common
-add-apt-repository -y ppa:webupd8team/java
 apt-get update
-echo "oracle-java8-installer shared/accepted-oracle-license-v1-1 select true" | debconf-set-selections
-apt-get install -y oracle-java8-installer
-apt-get install -y oracle-java8-set-default
-apt-get update && apt-get install -y maven
\ No newline at end of file
+sleep $[ ( $RANDOM % 10 )  + 1 ]s
```

Comment:
Please remove this

Reply:
happy to do it, however I noticed a lot less failed image build after adding that. I wonder if the fact that we send 20 requests at the same time for the same package might be a problem.